tasks/task01/narcissistic.py

```diff
@@ -1,20 +1,20 @@
 base = 3
 
 def digits(n):
-    # return list of digits of number
-    a = []
-    if n < 10:
-        a = [n]
-    else:
-        while n:
-            n, remainder = divmod(n, 10)
-            a.append(remainder)
-    return a
+  # return list of digits of number
+  a = []
+  if n < 10:
```

Comment:
There is no need in extra `if` branch, just `while` loop is sufficient. Try to analyse yourself